aptos-move/framework/aptos-experimental/sources/trading/market/order_placement.move

```diff
@@ -624,6 +629,34 @@ module aptos_experimental::order_placement {
         time_in_force: TimeInForce,
         fill_sizes: &mut vector<u64>,
     ): (Option<OrderCancellationReason>, CallbackResult<R>) {
+        let dead_mans_switch_enabled = market.is_dead_mans_switch_enabled();
+        if (dead_mans_switch_enabled) {
+            if (!is_order_valid(market.get_dead_mans_switch_tracker(), user_addr, timestamp::now_seconds())) {
```

Comment:
seems unnecessary to call timestamp::now_seconds() before seeing if user_addr has enabled dead man switch?